chart/slackconnector/templates/application.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Release.Name }}-app
```

Comment:
We already know that this is Application based on a Kind. I would suggest using only `{{ .Release.Name }}`